delivery_seur/views/stock_picking_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<!-- Copyright 2020 Tecnativa - David Vidal
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).-->
+    <record id="view_picking_withcarrier_out_form" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="delivery.view_picking_withcarrier_out_form"/>
+        <field name="groups_id" eval="[(4, ref('base.group_no_one'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='extra']" position="inside">
+                <group string="SEUR Technical">
```

Comment:
grupo técnico?

Reply:
Está a nivel de vista

Reply:
¿por cada carrier se tiene pensado crear nuevos campos "técnicos" en el picking? ¿no creéis que es mejor llevar estos campos a un módulo genérico del que dependan estos?

Reply:
@rlizana lo llegamos a pensar, pero debido a diversas consideraciones, lo hemos deshechado por el momento:

- Esta información es de bastante depuración
- No tiene intención de ser agregada
- Los campos vacíos en PG no suponen penalización de espacio (simplemente tenemos la cuestión de campos en el modelo).
- Añadir una dependencia ahora con el módulo fusionado es una incomodidad para la base de usuarios.
- Tener un módulo con solo 2 campos extra parece un poco vacuo (no es lo mismo que el `delivery_state` que sí que añade lógica asociada).

¿lo ves como algo bloqueante?

Reply:
@pedrobaeza no es bloqueante, era una sugerencia.